canonical-websites/www.ubuntu.com
# Updated redirect for /community/conduct

## Done

- Updated redirect for /community/conduct

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/community/conduct](https://0.0.0.0:8001/community/conduct) and see that you are redirected to  `/about/about-ubuntu/conduct`

## Issue / Card

Fixes #2598
